accepted/PSR-15-request-handlers.md

```diff
@@ -6,7 +6,7 @@ This document describes common interfaces for HTTP server request handlers
 that use HTTP messages as described by [PSR-7][psr7] or subsequent
 replacement PSRs.
 
-HTTP request handlers are a fundamental part of any web application. Server side
+HTTP request handlers are a fundamental part of any web application. Server-side
```

Comment:
Not sure I agree with this change. While hyphenating might be more precise, I have seen it with and without the hyphen. Would this modification really make the document better?

Reply:
Just something I saw along the way by going quite far into typos.
Answer is here, I believe: https://en.wikipedia.org/wiki/Server-side

Reply:
It's compound when used as a concept: "server-side".
It's separated when used as two words: "It's a concern that happens to be on the server('s) side.".